workbench/urls.py

```diff
@@ -61,7 +61,7 @@
         name='reset_state'
     ),
 
-    url(r'^admin/', include(admin.site.urls)),
+    url(r'^admin/', admin.site.urls),
```

Comment:
Why was this change needed?  I didn't think `include()` had been deprecated.

Reply:
it was causing some namespace issues in later versions of django22. As this was changed 1.9 onwards so removed it.

Reply:
Ah, I'd missed/forgotten that change specifically for Django admin URLs.